bridges/modules/xcm-bridge-hub-router/src/lib.rs

```diff
@@ -30,6 +30,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use bp_xcm_bridge_hub_router::MINIMAL_DELIVERY_FEE_FACTOR;
```

Comment:
@serban300 do we want to keep `MINIMAL_DELIVERY_FEE_FACTOR`, can we just remove it and use default one? wdyt?

Reply:
oh, I see, maybe we could get rid of `Default` impl for `BridgeState` (as follow-up or as a part of the big congestion PR)

Reply:
Yes, perfect. We can try to do that.